### DISCUSSION TITLE
Kotori greets independent IoT systems

### DISCUSSION THREAD

**Discussion by amotl** [daq-tools/kotori#44] [Show and tell]
Hi there,

this is a thread dedicated to all people building independent IoT systems who want to easily add a data historian to their setup. We are looking forward to nice and fruitful discussions here.

With kind regards,
Andreas.

**Comment by amotl** [comment]
Dear @jwillmer,

our ways crossed at https://github.com/earthobservations/wetterdienst/issues/342 the other day. Thanks!

Now that I am reading about your nice tutorials at [1,2,3], I want to humbly make you aware about Kotori. This piece of software might well resonate with your setup and will be able to add a long-term storage component for metrics / measurements to complement the ePaper module for local display.

While I recognized you are not a Python guy (yet), Kotori is really easy to operate and will add a data historian feature without further ado, being based on the best of breed of open source components you can get these days. As I see you already might know your way around MQTT [4], this won't be any stranger to you if you want to use that as a telemetry transport (HTTP will also work).

So, enjoy reading the documentation [5] and don't hesitate to get back to me with further questions by replying to this comment.

With kind regards,
Andreas.

[1] https://jwillmer.de/blog/projects/independent-iot-system-part-1
[2] https://jwillmer.de/blog/projects/independent-iot-system-part-2
[3] https://jwillmer.de/blog/projects/independent-iot-system-part-3
[4] https://jwillmer.de/blog/tools/raspberry-pi-zero-cc2531-case
[5] https://getkotori.org/

**Reply by jwillmer** [reply]
Cool, I will definitely check it out! The project from my blog is currently postponed since I'm investigating (building) wind generators for a second power source of the IoT system. But when I'm done with it I will play around with this project 👍
